build/rpm/main.go

```diff
@@ -162,7 +162,7 @@ Summary:            {{ .Summary }}
 
 Group:              Application/blockchain
 License:            GNU
```

Comment:
It's not directly related to this PR.

```suggestion
License:            LGPL-3.0
```